src/sentry/tagstore/snuba/backend.py

```diff
@@ -140,11 +140,23 @@ def __get_tag_key_and_top_values(self, project_id, group_id, environment_id,
 
     def __get_tag_keys(self, project_id, group_id, environment_id, limit=1000, keys=None):
         start, end = self.get_time_range()
+        return self.__get_tag_keys_for_projects(
+            [project_id],
+            group_id,
+            [environment_id] if environment_id else None,
+            start,
+            end,
+            limit,
+            keys,
+        )
+
+    def __get_tag_keys_for_projects(self, projects, group_id,
```

Comment:
Why two leading `_`? :bike:

Reply:
that seemed to be convention for this class